pkg/accountdb/dbo/user.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package dbo
+
+import (
+	"github.com/skyrings/skyring-common/tools/uuid"
```

Comment:
Let's use a smaller package, something like https://github.com/gofrs/uuid.

Reply:
this package is already added by someone

Reply:
Hmm, really? Ah, nevermind then.